chimaera.xml

```diff
@@ -1914,6 +1914,18 @@ Law of total tricks.
   <bid id="28">
     <value>3d</value>
     <desc>16+ points, 5+ diamonds.</desc>
+	
+	<condition type="include">
```

Comment:
Please keep the condition element in line with the <value> and <desc> elements - helps keep track of which elements are at which level.